include/libtorrent/kademlia/dht_tracker.hpp

```diff
@@ -111,7 +111,8 @@ namespace libtorrent { namespace dht {
 			, std::function<void(std::vector<tcp::endpoint> const&)> f);
 
 		void sample_infohashes(udp::endpoint const& ep, sha1_hash const& target
-			, std::function<void(time_duration
+			, std::function<void(sha1_hash
```

Comment:
iirc, there's an alias called `dht::node_id` which I think would be better suited here. They are both the same underlying type, but I think it would be better if they were distinct. at least in the API

Reply:
changed